src/constants/translations/en/common.json

```diff
@@ -10,13 +10,16 @@
   "finish": "Finish",
   "search": "Search",
   "delete": "Delete",
+  "showMore": "Show more",
+  "showLess": "Show less",
   "confirmButton": "OK",
   "uah": "UAH",
   "hour": "HOUR",
   "hourSlash": "/ HOUR",
   "beginner": "Beginner",
   "offers": "offers",
   "popularCategories": "Popular Categories",
+  "aboutOffer": "About Offer",
```

Comment:
isn't it needed to add aboutOffer also to ua translations

Reply:
Didn`t see it, fixed